package.json

```diff
@@ -12,6 +12,7 @@
     "url": "https://github.com/pugjs/pug.git"
   },
   "scripts": {
+    "format": "prettier --ignore-path .gitignore --write './**/*.js'",
```

Comment:
i did not feel comfortable formatting anything else, especially since even formatting `js` files required me to update a few mocks